app/controllers/ui_job_wizard_controller.rb

```diff
@@ -8,6 +8,14 @@ def categories
     render :json => {:job_categories =>job_categories}
   end
 
+  def template
+    job_template = JobTemplate.find_by(id: params[:id])
```

Comment:
Probably an edge case, but we should check the user has necessary permissions.

Reply:
@adamruzicka is there a before_action trick for that?

Reply:
Yes and no. Usually a controller is centered around a resource and then you can use such tricks to reduce code duplication. However in our case, the controller is not really centered around a resource (template in this case), so just doing `JobTemplate.authorized.find_by(...)` would be fine.

Also, `find_by` returns `nil` if no resource is found, don't we want `find` which raises an exception if the resource cannot be found?